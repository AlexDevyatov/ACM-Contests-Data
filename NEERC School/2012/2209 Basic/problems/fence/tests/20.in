12
-44 34
-40 -40
-37 -43
-31 -44
-22 -44
33 -40
37 -38
39 -27
40 12
31 27
14 37
-33 41
36 18
