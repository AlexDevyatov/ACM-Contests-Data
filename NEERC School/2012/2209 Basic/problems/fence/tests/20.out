13
-44 34
-40 -40
-37 -43
-31 -44
-22 -44
33 -40
37 -38
39 -27
40 12
36 18
31 27
14 37
-33 41
